libsplinter/src/service/scabbard/state.rs

```diff
@@ -358,10 +358,14 @@ impl From<InvalidTransactionResult> for InvalidTransaction {
 pub struct BatchInfo {
     pub id: String,
     pub status: BatchStatus,
-    #[serde(skip_serializing)]
+    #[serde(skip, default = "get_time_now")]
```

Comment:
Can you just use `"System::now"` as this value? Also, probably a good idea to use `skip_deserializing`

Reply:
I can use `System::now` I'll make that change. `skip` skips both serialization and deserialization.  So I thought it was the best option to keep the logic of the original `skip_serializing` while also doing the `skip_deserializing`. 

https://serde.rs/field-attrs.html#skip